plugins/must-gather/skills/must-gather-analyzer/scripts/analyze_pods.py

```
@@ -0,0 +1,224 @@
+    init_container_statuses = status.get('initContainerStatuses', [])
+
+    # Calculate ready containers
+    total_containers = len(spec.get('containers', []))
+    ready_containers = sum(1 for cs in container_statuses if cs.get('ready', False))
+
+    # Get overall phase
+    phase = status.get('phase', 'Unknown')
+
+    # Determine more specific status
+    pod_status = phase
+    reason = status.get('reason', '')
```

Ruff (0.14.4)

Local variable `reason` is assigned to but never used

Remove assignment to unused variable `reason`

(F841)